index.html

```diff
@@ -6883,6 +7140,303 @@ <h2 id="MediaStreamAudioSourceNode">
+          <a>Control message</a> passing. Communication
+          in the other direction is done using regular event loop tasks.
+        </p>
+        <p>
+          Each <a>AudioContext</a> has a single <dfn>control message
+          queue</dfn>, that is a list of <dfn data-lt="control message">control
+          messages</dfn>, that are a operations running on the <a>control
+          thread</a>.
+        </p>
+
+        <p>
+          <dfn id="queuing">Queuing an control message</dfn> means adding the
+          message to the end of the <a>control message queue</a> of an
+          <a>AudioContext</a>.
+        </p>
+        <p>
+          <a>Control messages</a> in a <a>control message queue</a> are ordered by
+          date of insertion. The <dfn>oldest message</dfn> is therefore the one
+          at the front of the <a>control message queue</a>.
+        </p>
+        <p>
+          <dfn data-lt="swap">Swapping</dfn> a <a>control message queue</a>
+          <var>Q<sub>A</sub></var> with another <a>control message queue</a>
+          <var>Q<sub>B</sub></var> means executing the following steps:
+        </p>
+        <ol>
+          <li> Let <var>Q<sub>C</sub></var> be a new, empty <a>control message
+                queue</a>. </li>
+          <li> Move all the <a>control messages</a> <var>Q<sub>A</sub></var> to
+              <var>Q<sub>C</sub></var>.</li>
+          <li> Move all the <a>control messages</a> <var>Q<sub>B</sub></var> to
+              <var>Q<sub>A</sub></var>.</li>
+          <li> Move all the <a>control messages</a> <var>Q<sub>C</sub></var> to
+              <var>Q<sub>B</sub></var>.</li>
+        </ol>
+        <p class="note">
+          For example, successfuly calling <code>start()</code> on an
+          <a>AudioBufferSourceNode</a> <code>source</code> adds a <a>control
+          message</a> to the <a href="#control-message-queue">control message
+          queue</a> of the <a>AudioContext</a> <code>source.context</code>.
+        </p>
+      </section>
+      <section>
+        <h3>
+          Asynchronous operations
+        </h3>
+        <p>
+          Calling methods on <a>AudioNode</a>s MUST to be done in two phases, a
+          synchronous operation and an asynchronous operation. For each method,
+          some part of the execution happens on the <a>control thread</a> (for
+          example, throwing an exception in case of invalid parameters), and
+          some part happens on the <a>rendering thread</a> (for example,
+          changing the value of an <a>AudioParam</a>).
+        </p>
+        <p>
+          In the description of each operation on <a>AudioNode</a>s and
+          <a>AudioContext</a>s, the synchronous section is marked with a ⌛. All
+          the other operations are executed <a href=
+          "https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel">
+          in parallel</a>, as described in [[HTML]].
+        </p>
+        <p>
```

Comment:
This describes asynchronous operations, that are implemented internally using a synchronous and an asynchronous part. I've tried to be a bit more clear about it.